modules/renter/hostdb/scan.go

```diff
@@ -242,40 +250,52 @@ func (hdb *HostDB) managedScanHost(entry modules.HostDBEntry) {
 	hdb.mu.Unlock()
 }
 
+// runs scan on a host
+func (hdb *HostDB) threadedProbeHost(hostEntry modules.HostDBEntry) {
```

Comment:
The comment on a function needs to be a full sentence, and needs to take the form 'funcName blah blah blah...'